modules/bench/README.adoc

```diff
@@ -33,3 +33,11 @@ All run through scripts in `./bin`:
 - `run-bench.sh ts-devices [--size <small|med|big>] [--count 1]` - kicks off an ECS job.
 
 You can also run it locally with (e.g.) `clojure -Mxtdb:4gb:bench-tpch --scale-factor 0.05` from the top-level of the core2 repo.
+
+
+=== Promotheus and grafana
```

Comment:
nit: adoc idiom is [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line), makes git diffs easier to read/work with in future.

another nit: aware it's internal docs but worth giving them a bit more structure - this reads a little like a brain dump. have a think on what the reader will need to extract from this :slightly_smiling_face:

Reply:
Reworded a bit.